DevEdu.API/DevEdu.Business.Tests/Group/GroupServiceTests.cs

```diff
@@ -0,0 +1,125 @@
+﻿using DevEdu.Business.Services;
+using DevEdu.DAL.Models;
+using DevEdu.DAL.Repositories;
+using Moq;
+using NUnit.Framework;
+using System.Collections.Generic;
+
+namespace DevEdu.Business.Tests.Group
+{
+    public class GroupServiceTests
+    {
+
+        private Mock<IGroupRepository> _mock;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _mock = new Mock<IGroupRepository>();
+        }
+
+        [TestCaseSource(typeof(GroupServiceExpecteds), nameof(GroupServiceExpecteds.AddGroup))]
```

Comment:
please use the approach that all others are using already
(you need to create GroupTestData static class with static methods that return some particular mock)